src/Compilers/CSharp/Desktop/CommandLine/CommandLineParser.cs

```diff
@@ -587,7 +595,18 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                                 }
                                 else
```

Comment:
The diffing certainly leaves a lot to be desired :) Looks like I have to either view the full file without diffs or create a branch of your changes to figure out which if this else corresponds to.